### PR TITLE
numpy: initial integration

### PR DESCRIPTION
**Pull request by DavidKorczynski**
Initial integration of numpy. Thread on the numpy mailing list discussing integrating with OSS-Fuzz https://mail.python.org/archives/list/numpy-discussion@python.org/thread/MR2BKUFZMGIB7IJOK4ME7Q46VUVEV3BT/

### CODE REVIEW

**Comment by DavidKorczynski** [comment]
https://github.com/google/oss-fuzz/pull/7680 is merged -- I will close/open this PR in a day or two when the base images are rebuild

File: projects/numpy/project.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+homepage: "numpy.org"
+main_repo: "https://github.com/numpy/numpy"
+language: python
+primary_contact: "david@adalogics.com"
```
Comment (jonathanmetzman) [inline projects/numpy/project.yaml:4]: no maintainers agreed?

**Comment by DavidKorczynski** [comment]
Mailing list thread discussing the OSS-Fuzz integration: https://mail.python.org/archives/list/numpy-discussion@python.org/thread/MR2BKUFZMGIB7IJOK4ME7Q46VUVEV3BT/ 

The discussion is moving but am still awaiting confirmation from maintainers.

File: projects/numpy/project.yaml (hunk 1 above)
Comment (DavidKorczynski) [reply projects/numpy/project.yaml]: Done!

**Review by oliverchang — APPROVED** [review]

**Pull request merged**